tap_helpscout/streams/teams.py

```diff
@@ -0,0 +1,16 @@
+from abc import ABC
+
+from .abstract import IncrementalStream
+
+
+class Teams(IncrementalStream, ABC):
```

Comment:
The same is true of the two above stream.py files, but why do we need to pass ABC to this class? Wouldn't it be easier across the entire integration if we were to make ABC an argument on BaseStream in abstract.py so that any classes inheriting from BaseStream also inherit ABC?

Reply:
removed ABC from here and kept in BaseStream class